examples/text-classification/run_glue.py

```diff
@@ -150,10 +150,11 @@ def main():
 
     def build_compute_metrics_fn(task_name: str) -> Callable[[EvalPrediction], Dict]:
         def compute_metrics_fn(p: EvalPrediction):
+            preds = p.predictions[0] if type(p.predictions) == tuple else p.predictions
```

Comment:
```suggestion
            preds = p.predictions[0] if isinstance(p.predictions, tuple) else p.predictions
```
(More pythonic since it will catch the subclasses too.)